subgraph.template.yaml

```diff
@@ -24,7 +24,7 @@ dataSources:
         - name: Rentals
           file: ./abis/Rentals.json
       eventHandlers:
-        - event: RentalStarted(address,uint256,address,address,address,uint256,uint256,address)
+        - event: RentalStarted(address,uint256,address,address,address,uint256,uint256,bool,address,bytes)
```

Comment:
Event is now called `AssetRented`

Reply:
https://github.com/decentraland/rentals-contract/blob/main/contracts/Rentals.sol#L104